app/views/groups/index.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="top_bar">
+  <%= render partial: 'shared/menu', locals: { title: 'GROUPS', menu_path: root_path } %>
+</div>
+
+<section class="groups_page">
+  <div>
+    <% @groups.each do |group| %>
+    <div class="group">
+     <div>
+      <a href=<%= group_expenses_path(group_id: group.id) %>>
+        <% if group.icon.attached?%>
+          <%= image_tag(group.icon) %>
+        <%else%>
+          <%= image_tag('group.png') %>
+        <%end%>
+        <!-- <%= image_tag(group.icon.attached? ? group.icon : 'group.png') %> -->
+        <div>
+          <h2><%= group.name %></h2>
+          <% if Time.now - group.created_at <= 1.minute %>
+            <p><%= time_ago_in_words(group.created_at, include_seconds: true) %> ago</p>
+          <% elsif Time.now - group.created_at <= 24.hours %>
+            <p><%= time_ago_in_words(group.created_at) %> ago</p>
+          <% else %>
+            <p><%= group.created_at.strftime('%b %d, %Y') %></p>
+          <% end %>
+        </div>
+        </a>
+      </div>
+      <div class="list_gen">
+        <h3>$ <%= group.expenses.sum(:amount) %></h3>
+        <div class="btn_list">
+          <button class="list_btn"><%= link_to 'Detail', group_path(id: group.id) %> </button>
```

Comment:
- [ ] Great job working on the design I think it is very great but I just have a samll note kindly align the button to be in the botton as shown in the original design since in your design it is just right after the items 🤩💯♻ so kindly align it in the bottom 

- Your design
![image](https://user-images.githubusercontent.com/58553711/221380309-053344ad-7ecf-471b-84a1-485b25cd0588.png)

- Original design
![image](https://user-images.githubusercontent.com/58553711/221380354-ba6989eb-365f-4564-a718-9d731fb0d020.png)